firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift

```diff
@@ -761,6 +900,32 @@ class CreditCardsTests: BaseTestCase {
         saveButton.waitAndTap()
     }
 
+    private func addCreditCard_TAE(name: String, cardNumber: String, expirationDate: String) {
+        tapCardName()
+        typeCardName(name: name)
+        tapCardNr()
```

Comment:
I know this is not created in this PR, but in oder to make tests clearer this should be tapCardNumber, similar to tapCardName